lgc/include/lgc/state/TargetInfo.h

```diff
@@ -37,15 +37,31 @@ namespace lgc {
 // Represents graphics IP version info. See https://llvm.org/docs/AMDGPUUsage.html#processors for more
 // details.
 struct GfxIpVersion {
-  unsigned major;    // Major version
-  unsigned minor;    // Minor version
-  unsigned stepping; // Stepping info
+  static const unsigned Unspecified = ~0U;
```

Comment:
Why is this changing inside LGC? What is the intended behavior if you use "Unspecified" for minor or stepping in the LGC interface when it wants to generate some code that depends on the gfxip version?

Reply:
Oh I see; you're not intending to use "Unspecified" in the gfxip being compiled, only in what you are comparing it against.

A helper like isGfx(major, minor), perhaps as a method on GfxIpVersion, might be clearer.

Reply:
OK. I will close this PR and make new changes. Thank you.